include/crm/crm.h

```diff
@@ -66,7 +66,7 @@ extern "C" {
  * >=3.0.13: Fail counts include operation name and interval
  * >=3.2.0:  DC supports PCMK_LRM_OP_INVALID and PCMK_LRM_OP_NOT_CONNECTED
  */
-#  define CRM_FEATURE_SET		"3.8.0"
+#  define CRM_FEATURE_SET		"3.9.0"
```

Comment:
Is it really necessary to bump CRM_FEATURE_SET?  3.8.0 was just done on March 29 and it hasn't been in any public releases yet.

Reply:
I'm in favor of always bumping it per PR. That makes it more obvious in git history everything that went into a particular feature set bump (important for backports), lets external code like resource agents check for a specific supported feature (also important for backports), and makes conflicts more obvious if multiple PRs are in progress that bump it. Plus we aren't going to run out of integers :)

There are no inefficiencies for multiple bumps in a single release, it's always just a lt/gt comparison.

Reply:
Okay, that makes sense.

Reply:
Okay, that makes sense.